src/components/Card/HomeCard.js

```diff
@@ -3,16 +3,20 @@ import React from "react";
 import { View, Text, StyleSheet, Pressable } from "react-native";
 import { useSelector } from "react-redux";
 
-import { Styles } from "../constants/Styles";
-import { selectIsLoggedIn } from "../redux/slices/authSlices";
+import { Styles } from "../../constants/Styles";
+import { selectIsLoggedIn } from "../../redux/slices/authSlices";
 
-const Card = ({ data, onButtonPress }) => {
+const HomeCard = ({ data, onButtonPress }) => {
   const isLoggedIn = useSelector(selectIsLoggedIn);
 
   const navigation = useNavigation();
   return (
     <Pressable
-      onPress={isLoggedIn ? () => navigation.navigate("Offers") : onButtonPress}
+      onPress={
+        isLoggedIn
+          ? () => navigation.navigate("OfferGeneration")
+          : onButtonPress
+      }
       style={styles.cardContainer}
     >
       <Text style={[Styles.mdSemiBold, styles.titleText]}>{data.title}</Text>
```

Comment:
InfoContainer can be a small component.

Reply:
HomeCard.js in itself is a small component, and this infoContainer is a non-reusable element. It is different for Offer page as well as Deals Page where different information is displayed in the card. So making a smaller component of it won't help with the reusability or enhancement of the code. If you still think it is a necessity to improve the code, please let me know and I will move ahead with it.

Reply:
Here itself it's reusable, in two places we can use it.